src/main/java/ch/njol/skript/expressions/ExprFuseDuration.java

```diff
@@ -0,0 +1,157 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.Timespan;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.entity.Creeper;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.TNTPrimed;
+import org.bukkit.entity.minecart.ExplosiveMinecart;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Fuse Duration")
+@Description({
+	"Sets or gets the fuse duration of an entity.",
+	"Supports Primed TNT, Creepers, and TNT Minecarts.",
+	"For Creepers, you can also retrieve their maximum fuse time."
+})
+@Examples({
+	"set fuse duration of {_tnt} to 4 seconds",
+	"set max fuse duration of last spawned creeper to 3 seconds",
+	"broadcast remaining fuse time of {_tnt}",
+})
+@Since("INSERT VERSION")
+public class ExprFuseDuration extends SimplePropertyExpression<Entity, Timespan> {
+
+	static {
+		register(ExprFuseDuration.class, Timespan.class, "[remaining|:max[imum]] fuse [duration|time]", "entities");
```

Comment:
This looks like it's a breaking change from the old creeper `fuse tick[s]` pattern, do you think it would be worth keeping it around as an optional version (for the sake of compatibility) or should we just scrap it and put a warning in the patch notes?

Reply:
The old creeper syntax actually didn't even work (it wouldn't register due to a bad check) so luckily we can make "breaking" changes from it

Reply:
Oh great that's fine then.